src/main/java/aquality/appium/mobile/screens/screenfactory/ScreenPlatform.java

```diff
@@ -0,0 +1,14 @@
+package aquality.appium.mobile.screens.screenfactory;
+
+import aquality.appium.mobile.application.PlatformName;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface ScreenPlatform {
```

Comment:
should we name it just `Screen`? 
@ScreenPlatform(platform = PlatformName.ANDROID), looks like too much platform for me

Reply:
or smth like `ScreenType`